types/@heroicons/react/outline.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@heroicons/react/outline';
```

Comment:
Instead of defining types like this for library. Import the icons in the following way - https://www.reddit.com/r/react/comments/y9q3au/can_find_module_heroiconsreact_or_its/